WebPortal/Pages/Forms/Internal/OnboardingApplication.razor

```diff
@@ -116,19 +116,28 @@
         }
         var sectorId = _sectors.Where(s => s.Org_Name_E == _onboardingApplication.Client_Sector).First().Organization_ID;
         var branches = _branches.Where(r => r.Superior_OrgId == sectorId).Select(r=> r.Org_Name_E).ToList();
+        if (!branches.Any())
+        {
+            branches.Add("N/A");
+        }
+
         return branches;
     }
 
     private List<string> LoadDivisions()
     {
-        if (string.IsNullOrEmpty(_onboardingApplication.Client_Branch))
+        if (string.IsNullOrEmpty(_onboardingApplication.Client_Branch) || _onboardingApplication.Client_Branch == "N/A")
         {
-            return new List<string>();
+            return new List<string>() { "N/A" };
```

Comment:
Can you add a constant for the "N/A" just in case.

Reply:
Yes will do, Ty